rednotebook/util/markup.py

```diff
@@ -183,8 +186,6 @@ def get_markup_for_day(day, with_text=True, with_tags=True, categories=None, dat
 
 
 def _get_config(target, options):
-    is_exporting_to_file = options.get('export_to_file', False)
-
```

Comment:
The export_to_file flag is still used in the tests. Shouldn't we completely remove it? Can you grep for it, please?

Reply:
Good catch, done